<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDialog</class>
 <widget class="QWidget" name="AccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="go-previous">
         <normaloff>../../theShell/statuscenter/PulseaudioPane</normaloff>../../theShell/statuscenter/PulseaudioPane</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Account</string>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="mainContainer" native="true">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FocusBarrier" name="focusBarrier">
          <property name="text">
           <string notr="true">Barrier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PROFILE</string>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeUsernameButton">
          <property name="text">
           <string>Change Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeEmailButton">
          <property name="text">
           <string>Change Email Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SECURITY</string>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changePasswordButton">
          <property name="text">
           <string>Change Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setup2faButton">
          <property name="text">
           <string>Set up Two Factor Authentication</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logOutButton">
          <property name="text">
           <string>Log Out</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FocusBarrier" name="focusBarrier_2">
          <property name="text">
           <string notr="true">Barrier</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="GamepadHud" name="gamepadHud" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamepadHud</class>
   <extends>QWidget</extends>
   <header location="global">gamepadhud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusBarrier</class>
   <extends>QPushButton</extends>
   <header location="global">focusbarrier.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>focusBarrier</tabstop>
  <tabstop>changeUsernameButton</tabstop>
  <tabstop>changeEmailButton</tabstop>
  <tabstop>changePasswordButton</tabstop>
  <tabstop>setup2faButton</tabstop>
  <tabstop>logOutButton</tabstop>
  <tabstop>focusBarrier_2</tabstop>
  <tabstop>backButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
